sign_stage_rpms

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+. settings
+
+echo "./list_unsigned_rpms $1 $GPGKEY"
```

Comment:
As a debug advice: use `bash -xe ./sign_stage_rpms` to make it print exactly what it's doing.
```suggestion
```

And @evgeni will then suggest to change the header to
```bash
#!/bin/bash

set -e
```

Then you can simply use `bash -x ./sign_stage_rpms`.